src/DataFramesMeta.jl

```diff
@@ -405,9 +405,29 @@ end
 
 function _transform!(t::AbstractVector, first::AbstractVector, start::Int, 
                      g::GroupedDataFrame, v::Function, starts::Vector, ends::Vector)
+    @inline function fill_column_vec!(t::AbstractVector, out, startpoint::Int, endpoint::Int, len::Int)
```

Comment:
What I mean when I said you could move the code inside the function is that if you use `@inline`, you can just drop the function barrier and put the code directly in the parent function.

Reply:
It should probably still be a function, since we call it twice per `_transform!` function. Once for the first case and once for the rest. We need to call it twice because we have to compute `first` in `transform(::GroupedDataFrame,...)` so not treating the first case separately would require calculating `first` twice.

Reply:
Right. Should be OK as-is then.